### PR TITLE
feat(pci.data-processing): adding a page for spark notebook details

### PR DESCRIPTION
**Pull request by Dimfacion**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #DATATR-97 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

## Description

Add a page that shows the details of a notebook

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/notebook-details.html [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+            >
+            </oui-tile-definition>
+            <!--Editor-->
+            <oui-tile-definition
+                data-term="{{:: 'data_processing_notebooks_details_information_editor_label' | translate }}"
+            >
+                <oui-tile-description>
+                    <oui-button
+                        size="m"
+                        icon-right="oui-icon-external-link"
```
Comment (ClaveauAlex) [inline packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/notebook-details.html:52]:
```suggestion
                        data-size="m"
                        data-icon-right="oui-icon-external-link"
```

File: packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "data_processing_notebooks_details_dashboard_label": "Tableau de bord",
+  "data_processing_notebooks_details_information_label": "Information générales",
```
Comment (ClaveauAlex) [inline packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/translations/Messages_fr_FR.json:3]:
```suggestion
  "data_processing_notebooks_details_information_label": "Informations générales",
```
If you choose to put a plurial for "générales", I think you can add an other one for "informations".

File: packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/notebook-details.html (hunk 1 above)
Comment (Dimfacion) [reply packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/notebook-details.html]: Fixed :)

File: packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/translations/Messages_fr_FR.json (hunk 2 above)
Comment (Dimfacion) [reply packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/translations/Messages_fr_FR.json]: Fixed :)

**Review by ClaveauAlex — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/notebook-details.controller.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import { DATA_PROCESSING_NOTEBOOKS_STATUSES } from '../../data-processing.constants';
+
+export default class {
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/notebook-details.controller.js:3]: **nitpick:** Please use named classes.
```suggestion
export default class NoteBookDetailsCtrl{
```

File: packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  "data_processing_notebooks_details_lifecycle_label": "Lifecycle",
+  "data_processing_notebooks_details_lifecycle_status_label": "Statut",
+  "data_processing_notebooks_details_lifecycle_start_label": "Démarrer",
+  "data_processing_notebooks_details_lifecycle_stop_label": "Arrêter",
+  "data_processing_notebooks_details_lifecycle_delete_label": "Supprimer",
+  "data_processing_notebooks_details_lifecycle_last_start_date_label": "Démarré depuis le",
+  "data_processing_notebooks_details_lifecycle_elapsed_label": "Temps écoulé",
+  "data_processing_notebooks_details_resources_label": "Resources",
+  "data_processing_notebooks_details_resources_notebook_label": "Notebook",
+  "data_processing_notebooks_details_resources_notebook_NB1-1_label": "<strong>NB1-1 :</strong> 1 vCore CPU, 4 Gio memory.",
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/translations/Messages_fr_FR.json:20]: **nitpick:** `NB1-1 ` can be maintained as a constant and passed using `translate-values`. Thus avoiding html in translation files `<strong>`

File: packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/translations/Messages_fr_FR.json [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  "data_processing_notebooks_details_resources_cluster_label": "Cluster",
+  "data_processing_notebooks_details_resources_cluster_description": "Le dimensionnement du cluster se réalise par la sélection du kernel depuis JupyterLab.",
+  "data_processing_notebooks_details_resources_cluster_description_label": "<strong>{{clusterName}} :</strong> driver - {{driverCores}} vCore CPU / {{driverMemory}} Gio memory (+ {{driverMemoryOverhead}} Mio overhead), executor {{executorCores}} vCores CPU / {{executorMemory}} Gio memory (+ {{executorMemoryOverhead}} Mio overhead), {{executorNumber}} executors, {{htPrice}} &euro; HT ({{ttcPrice}} TTC) / min."
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/translations/Messages_fr_FR.json:23]: **nitpick:** Avoid using.html in translation files. In html,
`'clusterName': '<strong>' + template.name + '</strong>'`
```suggestion
  "data_processing_notebooks_details_resources_cluster_description_label": "{{clusterName}} : driver - {{driverCores}} vCore CPU / {{driverMemory}} Gio memory (+ {{driverMemoryOverhead}} Mio overhead), executor {{executorCores}} vCores CPU / {{executorMemory}} Gio memory (+ {{executorMemoryOverhead}} Mio overhead), {{executorNumber}} executors, {{htPrice}} &euro; HT ({{ttcPrice}} TTC) / min."
```

File: packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/notebook-details.scss [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+  padding-top: 0.313rem;
+  line-height: normal;
+  color: #999;
+  font-size: 1rem;
+  font-weight: 400;
+}
+
+.notebook-details__content {
+  margin-top: 2rem;
+}
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/notebook-details.scss:21]: **nitpick:** Avoid writing custom css. Try using utilities from bootstrap utilities and helpers from `ovh-ui-ki`
https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-helpers-colors--primary
https://getbootstrap.com/docs/4.0/utilities/flex/
https://getbootstrap.com/docs/4.0/utilities/spacing/

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/notebook-details.controller.js (hunk 3 above)
Comment (Dimfacion) [reply packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/notebook-details.controller.js]: Fixed :)

File: packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/translations/Messages_fr_FR.json (hunk 4 above)
Comment (Dimfacion) [reply packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/translations/Messages_fr_FR.json]: Fixed :)

File: packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/translations/Messages_fr_FR.json (hunk 5 above)
Comment (Dimfacion) [reply packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/translations/Messages_fr_FR.json]: Fixed :)

**Review by ClaveauAlex — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/notebook-details.html [hunk 7]
```diff
@@ -0,0 +1,174 @@
...
+            data-state="pci.projects.project.data-processing.notebooks.details"
+            data-state-params="{ notebookId: $ctrl.notebook.id }"
+            data-active
+            ><span
+                data-translate="data_processing_notebooks_details_dashboard_label"
+            ></span>
+        </oui-header-tabs-item>
+    </oui-header-tabs>
+</oui-header>
+<div class="notebook-details__content">
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/notebook-details.html:32]: **nitpick:** Bootstrap classes `col-sm`, `col-md`,`col-lg` and `col` can only exist as a directly child of an element with `.row`.  So please have a row and then size the columns using the helpers.

**Review by anooparveti — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**